ext/bcmath/libbcmath/src/doaddsub.c

```diff
@@ -179,7 +226,7 @@ bc_num _bc_do_sub(bc_num n1, bc_num n2, size_t scale_min)
 
 	/* If n1 has more digits than n2, we now do that subtract. */
 	if (diff_len != min_len) {
-		for (count = diff_len - min_len; count > 0; count--) {
+		for (size_t count = diff_len - min_len; count > 0; count--) {
```

Comment:
Why did you change this back to what it was previously? Did the change have negative performance impact?
In any case, I don't think changes like these belong to a SIMD PR anyway and should be done separately.

Reply:
Ah, indeed. Separate the commits and measure each one.

Reply:
I fixed it

Reply:
> Why did you change this back to what it was previously? Did the change have negative performance impact?

Combining EXPECTED with a comparison was faster than using ZEND_ASSERT.